sklearn/metrics/classification.py

```diff
@@ -221,8 +221,11 @@ def confusion_matrix(y_true, y_pred, labels=None, sample_weight=None):
 
     References
     ----------
-    .. [1] `Wikipedia entry for the Confusion matrix
+    .. [1] `Wikipedia entry for the Confusion matrix*
            <https://en.wikipedia.org/wiki/Confusion_matrix>`_
+                *NOTE: Different conventions exist for confusion matrix axes.
+                Hence, Wikipedia's example gives true class along axis=0, while 
```

Comment:
Wikipedia changes frequently. We can't say this about it.

Reply:
Ah, good point. Will remove Wikipedia reference.